srtcore/srt.h

```diff
@@ -772,6 +757,26 @@ SRT_API       SRTSOCKET srt_create_socket(void);
 
 // Group management
 
+// Stubs when off
+
+typedef struct SRT_SocketGroupData_ SRT_SOCKGROUPDATA;
```

Comment:
Maybe just `typedef void SRT_SOCKGROUPDATA` if `ENABLE_EXPERIMENTAL_BONDING` is not defined. Then the structure `SRT_SocketGroupData_` can still be
```c++
typedef struct SRT_SocketGroupData_ {
   // ..
} SRT_SOCKGROUPDATA;
```

Reply:
What's exactly the advantage, beside that the `SRT_SOCKGROUPDATA` will not be a unique type name?

Reply:
Now some undefined `struct SRT_SocketGroupData_` is used if bonding is disabled.

Reply:
Just like the `FILE` symbol in the standard C library.